### PR TITLE
BXMSPROD-1091: add mail sendings to web-publishing jobs

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[BXMSPROD-1091](https://issues.redhat.com/browse/BXMSPROD-1091)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins retest this

File: job-dsls/jobs/automatic_web_publishing.groovy [hunk 1]
```diff
@@ -88,16 +95,31 @@ for (reps in REPO_CONFIGS) {
...
+            cleanWs()
+            }                    
+        }
     }
 """
 
 
     pipelineJob("${folderPath}/${repo}-automatic-web-publishing") {
 
+        disabled()
```
Comment (mareknovotny) [inline job-dsls/jobs/automatic_web_publishing.groovy:121]: why disabled ?

Comment (mbiarnes) [reply job-dsls/jobs/automatic_web_publishing.groovy]: fixed

**Review by mareknovotny — APPROVED** [review]
lgtm

**Pull request merged**